server/builtin/info.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package builtin
+
+import (
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/config"
+)
+
+func (a *builtinApp) infoCommandBinding() apps.Binding {
+	return commandBinding("info", pInfo, "", "display Apps plugin info")
```

Comment:
This binding should probably have an empty form to work properly, unless we want to add some argument. Either way, even if we add arguments, they will be static forms probably, so they should be added to the binding.